recipes/libpgmath/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "libpgmath" %}
+{% set version = "2019.08.01" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/flang-compiler/flang/archive/7ab2da73bac1ab637210a6a78c167f7138512a99.tar.gz
+  sha256: e670ab95a01aea412a9a799aa98c724a7868a4627ab058d44dc8e5a9df781e15
+  patches:
+    - printf.diff
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python
+    - pip
+    - clangxx   # [not osx]
+    # - lit
```

Comment:
please remove comments

Reply:
If you don't mind, I'd like to keep this to remind me to fix the `lit` conda package on windows.

Reply:
Can you add an explanatory comment then, please?

Reply:
Sure. Done.

Reply:
Thanks!